Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -62,9 +63,9 @@
 - I provided principles to my team that allowed other engineers to navigate more decisions of their own, without relying on me as a sole decision maker.
 - I accurately documented the options considered in decision making to ensure that we did not re-tread the same ground next time.
 - I took ownership of my team's retrospective process, making sure everyone had a voice in how our team works and evolves together.
-- I performed interviews for engineering candidates, providing detailed and useful feedback 
+- I performed interviews for engineering candidates, providing detailed and useful feedback.
 - I confidently pitched an idea, positively influencing and convincing people to take decisive action.
-- I effectively steered technical and non-technical conversations to positive outcomes.
+- I convinced my peers of a technical decision through asynchronous discussion by effectively communicating my perspective and understanding. 
```

Comment:
I like this idea of being _more concrete_ with these examples 👍

This first concrete example is about convincing/influencing others, which is just one aspect of "steering". I wonder if adding a couple more on other aspects (e.g. on listening, and supporting) would help to round out the message we're sending here; something like:

1. I published an RFC on a technical approach and incorporated feedback from other teams, resulting in a better path forward for my team.
2. I supported a colleague's idea by articulating its benefits and addressing concerns, leading to its acceptance.

Reply:
This is great, I like the considerations to other parts of steering, and it helps dig into it a bit further than just surface level of convincing or influencing others. 

I was considering the first one a little bit, and I wonder if we can change the wording just a little bit to try and capture the part about engaging in discussions and helping set direction together. I'll go with it for now, but I'm interested in hearing if you think we've moved closer towards the message of steering : 

> I published an RFC on a technical approach and then engaged with feedback from other teams, incorporating the results of discussions to create a better path forward for my team.

The second point is well written already and a helpful example 👍